test/unit/oidc_jwt_token_validate_test.rb

```diff
@@ -0,0 +1,82 @@
+require 'test_helper'
+require 'ostruct'
+require 'openssl'
+require 'jwt'
+
+class OidcJwtTokenValidateTest < ActiveSupport::TestCase
+  context '#valid_jwt?' do
+    def setup
+      @jwk = JWT::JWK.new(OpenSSL::PKey::RSA.new(2048))
+      exp = Time.now.to_i + 4 * 3600
+      payload, headers = { "name": "jwt token", "iat": 1557224758, "exp": exp, "typ": "Bearer", "aud": "rest-client", "iss": "127.0.0.1"}, { kid: @jwk.kid }
+
+      @token = JWT.encode(payload, @jwk.keypair, 'RS512', headers)
+    end
+
+    test 'if valid jwk json is passed' do
+      stub_request(:get, Setting['oidc_jwks_url'])
+        .to_return(body: {"keys": [@jwk.export]}.to_json)
+      if OpenSSL::PKey::RSA.new.respond_to?(:set_key)
```

Comment:
IIUC this essentially makes all of these tests do nothing other than setup. Why not use `skip` instead? or wrap the whole context in a condition?